packages/mathtype-ckeditor4/test/unit-language-editorLanguageDefined.test.js

```diff
@@ -0,0 +1,46 @@
+// Import the CKEDITOR 4 instance to use it un the construction of the instance
+import * as CKEDITOR from '../node_modules/ckeditor4/ckeditor';
+import { CKEditor4Integration } from '../plugin.src'
+
+// Define a global integration to be assigned
+let integration;
+
+/**
+ * This test is focused on checking the mathtype getLanguage function through the
+ * editor language definition
+ */
+describe('CKEditor4: Test the language property. TAG = Language',
```

Comment:
Maybe we could use different descriptions for each of the tests, e.g.
"Test the language property when passing only the browser language"
"Test the language property when passing different browser and MathType parameters language"
etc.

Reply:
This type of description is in the test itself and the general description I have put is this so that they are not the same

Reply:
Okay, I took a second look at the test reports and indeed all the info is there. Great!